required/latex-lab/latex-lab-minipage.dtx

```diff
@@ -126,6 +126,12 @@
       \hsize\@tempdima
       \textwidth\hsize \columnwidth\hsize
       \@parboxrestore
+%    \end{macrocode}
+% minipages are perhaps used in environments where the tagging of paragraphs
+% has been altered. So we try to restore this. TODO: this should
+% perhaps be done in \cs{@parboxrestore} directly. 
+%    \begin{macrocode}
+      \UseTaggingSocket{para/restore}
```

Comment:
probably needs a % at the end of line

Reply:
it is an expl3 style, so % not needed (and there are \socket_use commands nearby). But looking at it I should change some of them to \UseTaggingSocket.

Reply:
I only noticed the percents nearby so thought is is not within L3 layer code. Maybe in that case cleaning up the code near by should be done at some point (clearl not urgent)